protocol/activity_center_persistence.go

```diff
@@ -301,6 +330,13 @@ func (db sqlitePersistence) buildActivityCenterQuery(tx *sql.Tx, cursor string,
 
 	}
 
+	switch read {
+	case activityCenterQueryParamsReadRead:
+		readWhere = "AND a.read = 1"
+	case activityCenterQueryParamsReadUnread:
+		readWhere = "AND NOT(a.read)"
+	}
+
```

Comment:
Your types don't have a predefined unknown type, so you may need a default case for uninitialised `activityCenterQueryParamsRead` vars.